pkg/components/images_test.go

```diff
@@ -24,13 +24,13 @@ var _ = Describe("No registry override", func() {
 		Expect(GetReference(ComponentCalicoNode, "", "")).To(Equal("docker.io/calico/node@" + ComponentCalicoNode.Digest))
 	})
 	It("should render a tigera image correctly", func() {
-		Expect(GetReference(ComponentTigeraNode, "", "")).To(Equal("gcr.io/unique-caldron-775/cnx/tigera/cnx-node@" + ComponentTigeraNode.Digest))
+		Expect(GetReference(ComponentTigeraNode, "", "")).To(Equal("quay.io/tigera/cnx-node@" + ComponentTigeraNode.Digest))
```

Comment:
Any reason these tests need to have hard coded paths for the registry? 

We basically have to switch them back and forth between releases. 

cc @tmjd

Reply:
No, I think we should fix those. We had some like that, they were fixed but then I guess we added more tests that had it hard coded again.